cdm/src/main/java/thredds/client/catalog/builder/CatalogBuilder.java

```diff
@@ -991,6 +988,7 @@ private Element readMetadataFromUrl(java.net.URI uri) throws java.io.IOException
       </xsd:choice>
 
       <xsd:element name="resolution" type="duration" minOccurs="0"/>
+      <xsd:element name="calendar" type="xsd:string" minOccurs="0"/>
```

Comment:
I thougth calendar as an attribute of `<timeCoverage>` instead of an element like the others (start, end, duration and resolution). The reason in my mind is that calendar value provides the semantic for the time/periods represented in the inner elements of timeCoverage.